packages/example/src/pages/guides/navigation/tabs.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Tabs
+tabs: ['Sidebar', 'Tabs']
+---
+
+<PageDescription>
+
+You have the option of adding tabs to your pages. This is really only recommended if you have many pages with the same tabs. If you just have one page a set of tabs, it might be better for discoverability to make those menu items.
```

Comment:
Just a suggestion to add clarity to this paragraph, totally don't have to do it!

```suggestion
You have the option of adding tabs to your pages. This is only recommended if you have several pages that will use the same tabs. If you only have one page with a set of tabs, it might be better for discoverability to change the tabs to menu items.
```

Reply:
>totally don't have to do it!

You're herby no longer required to tack this on to your suggestions. 

Thank you for the corrections!!